metal3-dev/local-cbo.sh

```diff
@@ -14,6 +14,17 @@ if [ ! -d $cbo_path ]; then
     exit 1
 fi
 
+# Make the TLS files available in the expected location on the present host
+mkdir -p /etc/cluster-baremetal-operator/tls
```

Comment:
It'd be nice if this could be overridden so the certs are in e.g OCP_DIR - but it looks like it's hard-coded in the CBO atm https://github.com/openshift/cluster-baremetal-operator/blob/master/main.go#L97